src/client/Microsoft.Identity.Client/Cache/CacheSessionManager.cs

```diff
@@ -45,7 +45,7 @@ public async Task<MsalAccessTokenCacheItem> FindAccessTokenAsync()
             return await TokenCacheInternal.FindAccessTokenAsync(_requestParams).ConfigureAwait(false);
         }
 
-        public Task<Tuple<MsalAccessTokenCacheItem, MsalIdTokenCacheItem, Account>> SaveTokenResponseAsync(MsalTokenResponse tokenResponse)
+        public Task<(MsalAccessTokenCacheItem accessCacheItem, MsalIdTokenCacheItem tokenCacheItem, Account account)> SaveTokenResponseAsync(MsalTokenResponse tokenResponse)
```

Comment:
Since the tuple is essentially a type, shouldn't the properties be named with uppercase? 

```csharp
var result = Save(...);
log(result.accessTokenCacheItem); // now
log(result.AccessTokenCacheItem); // preffered ?
```